{
  "name": "babel4catch",
  "version": "1.0.5",
  "description": "babel plugin for funtion to use try/catch",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [
    "babel-plugin",
    "try",
    "catch"
  ],
  "author": "xsf",
  "license": "ISC"
}
